manual_cases.md

```
@@ -0,0 +1,592 @@
+| Field | Value |
+|-------|-------|
+| **ID** | OCP-87356 |
+| **Version** | 4.22 |
+| **Assignee** | Nidan Gavali (ngavali) |
+| **Author** | Nidan Gavali (ngavali) |
+| **Importance** | Medium |
+| **Test Type** | Functional |
+
+#### Setup
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)